src/env/development.js

```diff
@@ -76,7 +77,9 @@ import { ClassificationMixed } from '../examples/classification_mixed';
  */
 // import { AllTypes } from "../examples/all_types";
 
-const data = Buckets;
+// CHANGEME: Edit the example you want below:
+//const data = RichTextHtml;
+const data = RichTextTable;
```

Comment:
For reviewer: (dev-only) this is for local webpack-dev-server to renders the component in Hot reload the component.

(And all files in `src/examples/**`)